stencil/src/index.html

```diff
@@ -29,12 +29,16 @@
 
         <pulumi-root></pulumi-root>
 
+        <p>uf</p>
+
+        <home-slots></home-slots>
+
         <pulumi-chooser type="language" options="typescript,python,go,csharp,fsharp,visualbasic"></pulumi-chooser>
 
         <pulumi-choosable type="language" value="typescript">
             <pre><code></code>I am some TypeScript.</pre>
             <pulumi-chooser type="os" options="macos,windows"></pulumi-chooser>
-            <pulumi-choosable type="os" value="macos"> Some info on MacOS. </pulumi-choosable>
+            <pulumi-choosable type="os" value="macos"> Some sinfo on MacOS. </pulumi-choosable>
```

Comment:
```suggestion
            <pulumi-choosable type="os" value="macos"> Some info on MacOS. </pulumi-choosable>
```
I'm guessing this was unintentional